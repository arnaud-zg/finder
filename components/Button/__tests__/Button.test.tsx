import React from 'react'
import renderer from 'react-test-renderer'
import * as ButtonIndex from '../'
import { Button } from '../'

describe('Button', () => {
  it('should take a snapshot of ButtonIndex', () => {
    expect(ButtonIndex).toMatchSnapshot()
  })

  it('renders correctly', () => {
    const testRenderer = renderer.create(<Button>Submit</Button>)
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
