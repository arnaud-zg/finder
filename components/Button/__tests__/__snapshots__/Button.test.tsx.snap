// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="fssyr3r"
  type="submit"
>
  Submit
</button>
`;

exports[`Button should take a snapshot of ButtonIndex 1`] = `
Object {
  "Button": [Function],
}
`;
